core/Client.php

```diff
@@ -389,6 +389,6 @@ private function parseExceptionResponse($e)
             );
         }
 
-        throw new ResponseException($body, $e->getResponse()->getStatusCode());
+        throw new ResponseException($e->getMessage(), $e->getResponse()->getStatusCode());
```

Comment:
If we couldn't find a ResponseStatus or Error in the body, then the body may not hold anything useful, and the exception may be from Guzzle. As a last resort, we should raise that Exception message.